bin/node/src/main.rs

```diff
@@ -1,12 +1,96 @@
-mod chain_spec;
-#[macro_use]
-mod service;
-mod aleph_cli;
-mod cli;
-mod command;
-mod commands;
-mod rpc;
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
I'm a tad confused about the licence header –  why do we need it here, but not elsewhere?

Reply:
I consulted it with @mike1729, who put it before, and for consistency I'm removing it